cronjobs/db_controller.py

```
@@ -0,0 +1,172 @@
+            raise ValueError(f"Missing database configuration: {', '.join(missing)}")
+
+    def connect(self):
+        try:
+            self.conn = mariadb.connect(
+                user=self.user,
+                password=self.password,
+                host=self.host,
+                port=self.port,
+                database=self.database,
+            )
+        except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)